docs/guides/magicgui.md

```diff
@@ -0,0 +1,530 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.8.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+(magicgui)=
```

Comment:
What does this do?

Reply:
that's a "[target header](https://jupyterbook.org/reference/cheatsheet.html#target-headers)" (used for external linking purposes).  I saw that @kne42 had used them in other files, so I followed suit ☺️